assets/js/main.js

```diff
@@ -52,3 +52,20 @@ $("#btn-modal-small").click(function() {
 $(".modal .close").click(function() {
   $(".modal").fadeOut();
 });
+
+
+// script to show and hide password
+$(".show-hide").click(function() {
```

Comment:
This code will not work in our react webapp, let us talk in order to refactorize it.

Reply:
Another option is that we remake this logic using React. In that case, this code is only for the library demo, and should not be here.

Reply:
This code is Jquery is only demonstrative to see how it has to work, it really should be done with React. We could remove it so that it does not interfere.

Reply:
> This code is Jquery is only demonstrative to see how it has to work, it really should be done with React. We could remove it so that it does not interfere.

No, it is not. [We are including this code](https://github.com/FromDoppler/doppler-webapp/blob/master/public/index.html#L47) in UI Library clients.

The demonstrative code should be in another place.

Reply:
Ok